Example/AblyChatExample/ContentView.swift

```diff
@@ -13,8 +9,10 @@ struct ContentView: View {
     @State private var reactions = ""
     @State private var newMessage = ""
     
+    private let chatClient: ChatClient = MockChatClient(realtime: MockRealtime.create(), clientOptions: .init())
```

Comment:
This creates a new chat client each time the `ContentView` struct is created — that is, very frequently. What was wrong with having it as `@State` as it was originally?

Reply:
I don't remember exactly why I decided to remove it as `State` property tbh. But maybe because those properties are usually used with binding sign `$` which is not going to happen to `chatClient` property.

Reply:
https://github.com/ably-labs/ably-chat-swift/pull/34/commits/f734a0e84545d09defc262d7603b1adcf84353bc